lib/services/pure_service.ts

```diff
@@ -60,7 +60,7 @@ export abstract class PureService<E = string> {
   * at a certain migratory state.
   */
   public async handleApplicationStage(_stage: ApplicationStage) {
-
+    console.log('Function handleApplicationStage not implemented');
   }
```

Comment:
Services don't have to implement this method so this could create unnecessary spam in the logs

Reply:
Same as other empty functions: https://github.com/typescript-eslint/typescript-eslint/blob/v3.10.1/packages/eslint-plugin/docs/rules/no-empty-function.md

I'm open to suggestion on how to change this

Reply:
In this case I would silence the warning with a comment explaining that it's an optional override 👍